2025-02-25 14:13:36,843 INFO: Iniciando aplicação NFCe [in c:\prototipo\backend\app\__init__.py:54]
2025-02-25 14:15:42,217 INFO: Iniciando aplicação NFCe [in c:\prototipo\backend\app\__init__.py:54]
2025-02-25 14:16:50,989 INFO: Iniciando aplicação NFCe [in c:\prototipo\backend\app\__init__.py:54]
2025-02-25 14:20:10,989 INFO: Iniciando aplicação NFCe [in c:\prototipo\backend\app\__init__.py:54]
2025-02-25 14:20:53,761 INFO: Iniciando aplicação NFCe [in c:\prototipo\backend\app\__init__.py:54]
2025-02-25 14:22:44,897 INFO: Iniciando aplicação NFCe [in c:\prototipo\backend\app\__init__.py:54]
2025-02-25 15:22:43,637 INFO: Iniciando aplicação NFCe [in c:\prototipo\backend\app\__init__.py:64]
2025-02-25 15:22:57,672 INFO: Iniciando aplicação NFCe [in c:\prototipo\backend\app\__init__.py:64]
2025-02-25 15:22:58,828 INFO: Iniciando aplicação NFCe [in c:\prototipo\backend\app\__init__.py:64]
2025-02-25 15:23:24,484 INFO: Autenticação na Nuvem Fiscal realizada com sucesso [in c:\prototipo\backend\app\services\nuvem_fiscal\client.py:72]
2025-02-25 15:23:27,295 ERROR: Erro ao cadastrar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in c:\prototipo\backend\app\services\nuvem_fiscal\client.py:127]
2025-02-25 15:23:27,307 ERROR: Resposta da API: {
    "error": {
        "code": "InvalidJsonProperty",
        "message": "Property \"telefone\" does not refer to a known property in type \"Empresa.DTO.TEmpresa\""
    }
} [in c:\prototipo\backend\app\services\nuvem_fiscal\client.py:129]
2025-02-25 15:23:27,308 ERROR: Erro ao criar empresa: Erro ao cadastrar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in c:\prototipo\backend\app\services\empresa_service.py:53]
2025-02-25 15:23:27,309 ERROR: Erro ao criar empresa: Erro ao cadastrar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in c:\prototipo\backend\app\routes\empresa.py:106]
2025-02-26 14:47:10,367 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 14:47:11,548 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 14:47:23,275 INFO: Autenticação na Nuvem Fiscal realizada com sucesso [in C:\prototipo\backend\app\services\nuvem_fiscal\client.py:72]
2025-02-26 14:47:26,193 ERROR: Erro ao cadastrar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in C:\prototipo\backend\app\services\nuvem_fiscal\client.py:127]
2025-02-26 14:47:26,193 ERROR: Resposta da API: {
    "error": {
        "code": "InvalidJsonProperty",
        "message": "Property \"telefone\" does not refer to a known property in type \"Empresa.DTO.TEmpresa\""
    }
} [in C:\prototipo\backend\app\services\nuvem_fiscal\client.py:129]
2025-02-26 14:47:26,194 ERROR: Erro ao criar empresa: Erro ao cadastrar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in C:\prototipo\backend\app\services\empresa_service.py:53]
2025-02-26 14:47:26,194 ERROR: Erro ao criar empresa: Erro ao cadastrar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in C:\prototipo\backend\app\routes\empresa.py:106]
2025-02-26 14:52:42,747 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 14:53:38,742 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 14:55:04,660 INFO: Autenticação na Nuvem Fiscal realizada com sucesso [in C:\prototipo\backend\app\services\nuvem_fiscal\client.py:72]
2025-02-26 14:55:08,791 ERROR: Erro ao cadastrar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in C:\prototipo\backend\app\services\nuvem_fiscal\client.py:127]
2025-02-26 14:55:08,791 ERROR: Resposta da API: {
    "error": {
        "code": "InvalidJsonProperty",
        "message": "Property \"telefone\" does not refer to a known property in type \"Empresa.DTO.TEmpresa\""
    }
} [in C:\prototipo\backend\app\services\nuvem_fiscal\client.py:129]
2025-02-26 14:55:08,792 ERROR: Erro ao criar empresa: Erro ao cadastrar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in C:\prototipo\backend\app\services\empresa_service.py:53]
2025-02-26 14:55:08,793 ERROR: Erro ao criar empresa: Erro ao cadastrar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in C:\prototipo\backend\app\routes\empresa.py:106]
2025-02-26 14:58:22,062 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 14:59:22,468 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 14:59:49,877 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:03:00,008 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:03:11,855 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:03:36,270 INFO: Autenticação na Nuvem Fiscal realizada com sucesso [in C:\prototipo\backend\app\services\nuvem_fiscal\client.py:75]
2025-02-26 15:03:39,108 ERROR: Erro ao cadastrar empresa: {
    "error": {
        "code": "InvalidJsonProperty",
        "message": "Property \"telefone\" does not refer to a known property in type \"Empresa.DTO.TEmpresa\""
    }
} [in C:\prototipo\backend\app\services\nuvem_fiscal\client.py:141]
2025-02-26 15:03:39,108 ERROR: Erro ao criar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in C:\prototipo\backend\app\services\empresa_service.py:53]
2025-02-26 15:03:39,110 ERROR: Erro ao criar empresa: 400 Client Error: Bad Request for url: https://api.nuvemfiscal.com.br/empresas [in C:\prototipo\backend\app\routes\empresa.py:106]
2025-02-26 15:09:32,905 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:10:29,072 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:10:43,871 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:10:45,072 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:10:53,178 INFO: Autenticação na Nuvem Fiscal realizada com sucesso [in C:\prototipo\backend\app\services\nuvem_fiscal\client.py:75]
2025-02-26 15:10:56,180 INFO: Empresa EMPREENDIMENTOS PAGUE MENOS S/A cadastrada com sucesso. [in C:\prototipo\backend\app\services\empresa_service.py:47]
2025-02-26 15:11:00,766 INFO: Empresa EMPREENDIMENTOS PAGUE MENOS S/A atualizada com sucesso. [in C:\prototipo\backend\app\services\empresa_service.py:104]
2025-02-26 15:11:02,834 INFO: Empresa EMPREENDIMENTOS PAGUE MENOS S/A desativada com sucesso. [in C:\prototipo\backend\app\services\empresa_service.py:218]
2025-02-26 15:11:05,284 INFO: Empresa EMPREENDIMENTOS PAGUE MENOS S/A excluída com sucesso. [in C:\prototipo\backend\app\services\empresa_service.py:189]
2025-02-26 15:50:36,386 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:50:59,414 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:53:30,561 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 15:54:49,354 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:64]
2025-02-26 16:27:30,914 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:27:32,096 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:29:29,504 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:31:07,453 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:32:54,807 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:33:10,867 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:33:27,997 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:34:40,515 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:48:39,713 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:48:41,174 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:51:31,043 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:51:44,991 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:52:09,597 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:52:10,992 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
2025-02-26 16:55:23,299 INFO: Iniciando aplicação NFCe [in C:\prototipo\backend\app\__init__.py:63]
